### PR TITLE
DATA: Create Cunstomer

### LINKED ISSUE #3: DATA: Create `Customer`

### PR DESCRIPTION
**Pull request by iamDiyorjon**
closes #3

### CODE REVIEW

**Review by zuhriddinzayniddinov — CHANGES_REQUESTED** [review]
Where is prop

File: DBShoping/Models/Customer.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{
+    internal class Customer
+    {
+        public Guid CustomerID { get; set; }
+        public string CompanyName { get; set; }
+        public string City { get; set; }
+        public string Region { get; set; }
+        public string Country { get; set; }
+        public string Phone { get; set; }
+
```
Comment (zuhriddinzayniddinov) [inline DBShoping/Models/Customer.cs]: Remove empty line

**Comment by iamDiyorjon** [comment]
closes #3

**Review by zuhriddinzayniddinov — CHANGES_REQUESTED** [review]
Remove empty line

File: DBShoping/Models/Customer.cs [hunk 2]
```diff
@@ -14,6 +14,5 @@ internal class Customer
         public string Region { get; set; }
         public string Country { get; set; }
         public string Phone { get; set; }
-
```
Comment (zuhriddinzayniddinov) [inline DBShoping/Models/Customer.cs]: ok

**Pull request merged**